includes/Infrastructure/ServiceContainer/SimpleServiceContainer.php

```diff
@@ -53,6 +53,10 @@ public function get( $id ): Service {
 
 		$service = $this->offsetGet( $id );
 
+		if ( ! $service instanceof Service ) {
```

Comment:
Can you explain this change and maybe by adding a unit test.

Reply:
`$service = $this->offsetGet( $id );` above calls `ArrayObject:offsetGet()`, which returns `false|mixed`. PHPStan complains about that because this method here obviously returns `Service` and nothing else.

This can be resolved by either a `@var Service $service` docblock or this simple check here.

In fact, I should probably do the former because it's not possible to put anything other than a Service into this.

Reply:
Addressed now.